src/cardano_common.h

```diff
@@ -0,0 +1,50 @@
+/*
+ * Common code for cracking Cardano 128-byte length legacy secret Keys (a.k.a XPrv).
+ *
+ * This software is Copyright (c) 2022, Pal Dorogi <pal dot dorogi at gmail.com>
+ * and it is hereby released to the general public under the following terms:
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted.
+ *
+ * This file takes replicated but common code, shared between the CPU
+ * and the GPU formats, and places it into one common location.
+ *
+ * This files is common for its future use, and currently there isn't an OpenCL format
+ * implementing this feature.
+ *
+ */
+
+#include "common.h"
+#include "formats.h"
+#include "jumbo.h"
```

Comment:
Out of these 3, probably only `#include "formats.h"` is needed here.

Reply:
I have removed all unnecessary includes.